leftwm/src/bin/leftwm-check.rs

```diff
@@ -279,6 +303,10 @@ fn check_theme_toml(filepath: PathBuf, verbose: bool) -> Result<PathBuf> {
                 if verbose {
                     println!("The theme file looks OK.");
                 }
+                println!(
+                    "\x1b[1;93mWARN: TOML as config format is about to be deprecated.
```

Comment:
this `\x1b[1;93` is for being printed out with a color right? Why not choosing a crate for doing this?

Reply:
Hm... I'm seeting that this is done in the other parts of this binary as well, I think an extra issue for this might be suitable.

Reply:
Also not sure if its worth pulling another crate for this niche use case. I tend to keep deps tight.

Reply:
> I tend to keep deps tight.

May I ask wht the reason is?

Reply:
I guess a slight tendency for minimalism. Not as dogmatic or strickt as the suckless guys, but you definitely want to keep an eye on the dep count.
If you carelessly add crates you easily might bloat your project. Maybe not the resulting binary size, but at least cargo updates and compilation time.

Reply:
> If you carelessly add crates you easily might bloat your project. Maybe not the resulting binary size, but at least cargo updates and compilation time.

I understand this point, but on the other hand I think that would be in contrast to the idea of having crates which should help you to reinvent the wheel.